src/Unity.Mathematics/affine_transform.cs

```diff
@@ -94,11 +94,13 @@ public AffineTransform(float4x4 m)
 
         /// <summary>Implicit float3x4 cast operator.</summary>
         /// <param name="m">The AffineTransform.</param>
```

Comment:
Thanks @unpacklo changes lgtm.
I'm wondering here if while we are making changes to this file, can we also add a _zero_ static readonly property for the affine_transform ? Like we have for the float4x4 ?

Reply:
Done!

Reply:
Thanks Dale!